docs/product-assembly_guided-decision-tables/src/main/asciidoc/main.adoc

```diff
@@ -1,4 +1,4 @@
-[id='assembly_guided-decision-tables']
+[id='assembly_building-custom-dashboard-widgets']
 
```

Comment:
assembly_building-custom-pages?

Reply:
highlighted text is assembly id, which is based on the epic title.
No change required. Left as is.